[2025-03-17T14:32:51.767-0400] {processor.py:186} INFO - Started process (PID=31997) to work on /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-17T14:32:51.767-0400] {processor.py:914} INFO - Processing file /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-03-17T14:32:51.767-0400] {logging_mixin.py:190} INFO - [2025-03-17T14:32:51.767-0400] {dagbag.py:588} INFO - Filling up the DagBag from /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-17T14:32:51.781-0400] {logging_mixin.py:190} INFO - [2025-03-17T14:32:51.780-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-03-17T14:32:51.781-0400] {logging_mixin.py:190} INFO - [2025-03-17T14:32:51.781-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-03-17T14:32:51.781-0400] {processor.py:927} WARNING - No viable dags retrieved from /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-17T14:32:51.811-0400] {processor.py:208} INFO - Processing /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.045 seconds
[2025-03-17T14:34:30.747-0400] {processor.py:186} INFO - Started process (PID=32155) to work on /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-17T14:34:30.747-0400] {processor.py:914} INFO - Processing file /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-03-17T14:34:30.748-0400] {logging_mixin.py:190} INFO - [2025-03-17T14:34:30.748-0400] {dagbag.py:588} INFO - Filling up the DagBag from /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-17T14:34:30.764-0400] {logging_mixin.py:190} INFO - [2025-03-17T14:34:30.764-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-03-17T14:34:30.764-0400] {logging_mixin.py:190} INFO - [2025-03-17T14:34:30.764-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-03-17T14:34:30.764-0400] {processor.py:927} WARNING - No viable dags retrieved from /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-17T14:34:30.794-0400] {processor.py:208} INFO - Processing /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.049 seconds
[2025-03-17T14:36:42.548-0400] {processor.py:186} INFO - Started process (PID=32318) to work on /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-17T14:36:42.548-0400] {processor.py:914} INFO - Processing file /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-03-17T14:36:42.549-0400] {logging_mixin.py:190} INFO - [2025-03-17T14:36:42.549-0400] {dagbag.py:588} INFO - Filling up the DagBag from /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-17T14:36:42.562-0400] {logging_mixin.py:190} INFO - [2025-03-17T14:36:42.562-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-03-17T14:36:42.563-0400] {logging_mixin.py:190} INFO - [2025-03-17T14:36:42.563-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-03-17T14:36:42.563-0400] {processor.py:927} WARNING - No viable dags retrieved from /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-17T14:36:42.592-0400] {processor.py:208} INFO - Processing /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.046 seconds
[2025-03-17T14:38:45.306-0400] {processor.py:186} INFO - Started process (PID=32504) to work on /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-17T14:38:45.307-0400] {processor.py:914} INFO - Processing file /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-03-17T14:38:45.307-0400] {logging_mixin.py:190} INFO - [2025-03-17T14:38:45.307-0400] {dagbag.py:588} INFO - Filling up the DagBag from /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-17T14:38:45.322-0400] {logging_mixin.py:190} INFO - [2025-03-17T14:38:45.322-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-03-17T14:38:45.322-0400] {logging_mixin.py:190} INFO - [2025-03-17T14:38:45.322-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-03-17T14:38:45.322-0400] {processor.py:927} WARNING - No viable dags retrieved from /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-17T14:38:45.352-0400] {processor.py:208} INFO - Processing /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.047 seconds
[2025-03-17T14:40:51.400-0400] {processor.py:186} INFO - Started process (PID=32702) to work on /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-17T14:40:51.401-0400] {processor.py:914} INFO - Processing file /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-03-17T14:40:51.401-0400] {logging_mixin.py:190} INFO - [2025-03-17T14:40:51.401-0400] {dagbag.py:588} INFO - Filling up the DagBag from /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-17T14:40:51.416-0400] {logging_mixin.py:190} INFO - [2025-03-17T14:40:51.415-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-03-17T14:40:51.416-0400] {logging_mixin.py:190} INFO - [2025-03-17T14:40:51.416-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-03-17T14:40:51.416-0400] {processor.py:927} WARNING - No viable dags retrieved from /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-17T14:40:51.448-0400] {processor.py:208} INFO - Processing /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.049 seconds
[2025-03-17T14:42:58.705-0400] {processor.py:186} INFO - Started process (PID=32883) to work on /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-17T14:42:58.705-0400] {processor.py:914} INFO - Processing file /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-03-17T14:42:58.706-0400] {logging_mixin.py:190} INFO - [2025-03-17T14:42:58.706-0400] {dagbag.py:588} INFO - Filling up the DagBag from /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-17T14:42:58.719-0400] {logging_mixin.py:190} INFO - [2025-03-17T14:42:58.719-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-03-17T14:42:58.719-0400] {logging_mixin.py:190} INFO - [2025-03-17T14:42:58.719-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-03-17T14:42:58.719-0400] {processor.py:927} WARNING - No viable dags retrieved from /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-17T14:42:58.749-0400] {processor.py:208} INFO - Processing /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.046 seconds
[2025-03-17T14:45:14.661-0400] {processor.py:186} INFO - Started process (PID=33065) to work on /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-17T14:45:14.662-0400] {processor.py:914} INFO - Processing file /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-03-17T14:45:14.662-0400] {logging_mixin.py:190} INFO - [2025-03-17T14:45:14.662-0400] {dagbag.py:588} INFO - Filling up the DagBag from /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-17T14:45:14.678-0400] {logging_mixin.py:190} INFO - [2025-03-17T14:45:14.677-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-03-17T14:45:14.678-0400] {logging_mixin.py:190} INFO - [2025-03-17T14:45:14.678-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-03-17T14:45:14.678-0400] {processor.py:927} WARNING - No viable dags retrieved from /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-17T14:45:14.709-0400] {processor.py:208} INFO - Processing /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.050 seconds
[2025-03-17T14:47:33.916-0400] {processor.py:186} INFO - Started process (PID=33241) to work on /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-17T14:47:33.917-0400] {processor.py:914} INFO - Processing file /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-03-17T14:47:33.917-0400] {logging_mixin.py:190} INFO - [2025-03-17T14:47:33.917-0400] {dagbag.py:588} INFO - Filling up the DagBag from /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-17T14:47:33.932-0400] {logging_mixin.py:190} INFO - [2025-03-17T14:47:33.931-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-03-17T14:47:33.932-0400] {logging_mixin.py:190} INFO - [2025-03-17T14:47:33.932-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-03-17T14:47:33.932-0400] {processor.py:927} WARNING - No viable dags retrieved from /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-17T14:47:33.961-0400] {processor.py:208} INFO - Processing /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.046 seconds
[2025-03-17T14:49:45.921-0400] {processor.py:186} INFO - Started process (PID=33437) to work on /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-17T14:49:45.922-0400] {processor.py:914} INFO - Processing file /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-03-17T14:49:45.923-0400] {logging_mixin.py:190} INFO - [2025-03-17T14:49:45.923-0400] {dagbag.py:588} INFO - Filling up the DagBag from /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-17T14:49:45.938-0400] {logging_mixin.py:190} INFO - [2025-03-17T14:49:45.937-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-03-17T14:49:45.938-0400] {logging_mixin.py:190} INFO - [2025-03-17T14:49:45.938-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-03-17T14:49:45.938-0400] {processor.py:927} WARNING - No viable dags retrieved from /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-17T14:49:45.969-0400] {processor.py:208} INFO - Processing /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.049 seconds
[2025-03-17T14:51:55.952-0400] {processor.py:186} INFO - Started process (PID=33721) to work on /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-17T14:51:55.952-0400] {processor.py:914} INFO - Processing file /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-03-17T14:51:55.953-0400] {logging_mixin.py:190} INFO - [2025-03-17T14:51:55.953-0400] {dagbag.py:588} INFO - Filling up the DagBag from /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-17T14:51:55.968-0400] {logging_mixin.py:190} INFO - [2025-03-17T14:51:55.967-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-03-17T14:51:55.968-0400] {logging_mixin.py:190} INFO - [2025-03-17T14:51:55.968-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-03-17T14:51:55.968-0400] {processor.py:927} WARNING - No viable dags retrieved from /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-17T14:51:56.005-0400] {processor.py:208} INFO - Processing /Users/nidhipatel/ApacheAirflow_Tutorial/py_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.055 seconds
